Doc update (#72)
DataDog/dd-sdk-android: docs/Migrating_To_1.0.0.md

            .setLogcatLogsEnabled(true)
            .build();
```

#### Attributes

> The attributes were created or removed with the `Logger.addField()` or `Logger.removeField()`

Reviewer: ```suggestion
In earlier versions, attributes were created or removed with the `Logger.addField()` or `Logger.removeField()`
```

> methods. These methods were rename for consistency purposes, and are now  `Logger.addAttribute()`

Reviewer: ```suggestion
methods. These methods were renamed for consistency purposes, and are now `Logger.addAttribute()`
```

>  and`Logger.removeAttribute()`, but they will behave the same way as the old ones.

Reviewer: ```suggestion
 and `Logger.removeAttribute()`. Their behavior remains the same.
```
